### ISSUE TITLE
Only top-most widget changes are being tracked

### ISSUE THREAD

**Issue by frankiejarrett** [wp-stream/stream#191]
When adding or removing any widget below the top-most one from a sidebar area the action is not logged in the activity stream.

![11b4d07a-5f19-11e3-899a-558bf395c93b](https://f.cloud.github.com/assets/522158/2052631/d5ae8042-8a99-11e3-9462-d363a1b33d4d.png)
1. Using the example screenshot seen above, when you drag the RSS widget out of the area it does not get recorded as deleted. But if you were to drag the Search widget out of the area, that would be tracked. This is because Search is the top-most widget, and actions for it appear to be firing just fine.
2. Similarly, I discovered that when widgets are dragged into the area (not into the top-most position) that it is not tracked as being added to the widget area.

So in conclusion, the problem seems to be that only the top-most widget is being tracked as added or removed.

This is a continuation of #33
